bundles/org.openhab.binding.gree/README.md

```diff
@@ -0,0 +1,139 @@
+# GREE Binding
+
+This binding integrates GREE Air Conditioners.
+
+Note : The GREE Air Conditioner must already be setup on the WiFi network and must have a fixed IP Address.
+
+## Supported Things
+
+This binding supports one Thing type `airconditioner`.
+
+## Discovery
+
+Once the GREE is on the network (WiFi active) it could be discovery automatically.
+An IP broadcast message is sent and every responding units gets added to the Inbox. 
+
+## Binding Configuration
+
+No binding configuration is required.
+
+#### Manual Thing Creation
+
+Fans can be manually created in the *PaperUI* or *HABmin*, or by placing a *.things* file in the *conf/things* directory.  See example below.
+
+
+## Thing Configuration
+
+The Air Conditioner's IP address is mandantory, all other parameters are optional. 
+If the broadcast is not set (default) it will be derived from openHAB's network setting (PaperUI:Configuration:System:Network Settings). 
+Change this only for good reasons.
+
+## Channels
+
+The following channels are supported for fans:
+
+| Channel Name  | Item Type | Description                                                                                       |
+|---------------|-----------|---------------------------------------------------------------------------------------------------|
+| power         | Switch    | Power on/off the Air Conditioner                                                                  |
+| mode          | String    | Sets the operating mode of the Air Conditioner                                                    |
+|               |           | Mode can be one of auto/cool/eco/dry/fan/turboheat or on/off                                      |
+|               |           | You could also send "0".."4", which will be send transparent to the device:                       |
+|               |           | those map to: "0"=Auto, "1"=Cool, "2"=Dry, "3"=Fan only, "4"=heat                                 | 
+|               |           | Check the Air Conditioner's operating manual for supported modes.                                 |
```

Comment:
Can you mention which of the three variants (auto/cool/eco/dry/fan/turboheat OR on/off OR 0-4) is used, when the status is read from the AC?

Reply:
those are text values, some of them map into multi parameters

Reply:
When the user monitors changing of the mode in a Rule, he must known which values he has to expect. E.g. if he checks for 0-4 it won't work, because the values published to the Channel are auto/cool/eco/dry/fan/turboheat. I would mention that auto/cool/eco/dry/fan/turboheat is the default. From the usability perspective it might be even better to remove 0-4 at all, to not to confuse the user. Or is there a use case I don't see?

Reply:
it's already like this :-) just updated the README